internal/hcsoci/hcsdoc_wcow.go

```diff
@@ -63,18 +61,26 @@ func createMountsConfig(ctx context.Context, coi *createOptionsInternal) (*mount
 					return nil, fmt.Errorf("failed to eval symlinks for mount source %q: %s", mount.Source, err)
 				}
 				mdv2.HostPath = src
+			} else if mount.Type == "virtual-disk" || mount.Type == "physical-disk" || mount.Type == "ExtensibleVirtualDisk" {
+
```

Comment:
Is it ok to eliminate the existing behavior of always checking a vSMB mount first regardless of mount.Type?

Reply:
I think so. We currently have a bug that doesn't allow using the same host path as both scsi and vsmb mount because of this.